Form layout—Documentation (HDS-4886) (#2947)
hashicorp/design-system: website/docs/components/form/layout/partials/specifications/anatomy.md

Author: Is this good now? @LilithJames-HDS updated it.

https://hds-website-git-kristin-hds-4886-form-layout-d-efa4db-hashicorp.vercel.app/components/form/layout?tab=specifications

## Anatomy

> ![Anatomy of formLayout](/assets/components/form/layout/form-anatomy-expanded.png)

Reviewer: In the image, "additional content" isn't a part of the Section Header, but it should be, correct?

> | Element                           | Usage                                                                                   |

Reviewer: Multi field group should be on this list as well

|-----------------------------------|-----------------------------------------------------------------------------------------|

> | Title/section title               | Optional                                                                                |

Reviewer 2: ```suggestion
| Title/Section title               | Optional                                                                                |
```

> | Description/section description   | Optional                                                                                |

Reviewer 2: ```suggestion
| Description/Section description   | Optional                                                                                |
```

| Section                           | May consist of multiple content types, including text, fields, and multi field groups.  |

> | Form error                        | Optional, uses the critical inline [Alert](components/alert#status-alerts)              |

Reviewer 3: ```suggestion
| Form error                        | Optional, uses the critical inline [Alert](/components/alert#status-alerts)              |
```
[Issue] broken link

> | Field                             | Required, supports any Form component or text elements                                  |

Reviewer: > or text element

?? Doesn't this just accept form components?

| Separator                         | Optional                                                                                |

> | Footer                            | Most often consists of [ButtonSet](components/button-set) and calls to action           |

Reviewer 3: ```suggestion
| Footer                            | Most often consists of [ButtonSet](/components/button-set) and calls to action           |
```
[Issue] broken link

> | Footer                            | Most often consists of [ButtonSet](/components/button-set) and calls to action           |

Reviewer: [nit] We often just refer to these as actions so they're not confused with more marketing type CTAs
```suggestion 
| Footer                            | Most often consists of [ButtonSet](/components/button-set) and actions           |
```

## Anatomy

<!-- image then table -->

> ![Anatomy of formLayout](/assets/components/alert/form-layout-anatomy-inline.png)

Author: @LilithJames-HDS Would you be able to add an image and the anatomy info for this section?
